tests/unit-tests/data_utils.py

```diff
@@ -343,6 +343,83 @@ def generate_dummy_homo_graph(size='tiny', gen_mask=True):
 
     return hetero_graph
 
+def generate_dummy_homo_failure_graph(size='tiny', gen_mask=True, type='node'):
+    """
+    generate a dummy homogeneous graph.
```

Comment:
Can you add more comments about what is the purpose of this dummy homo failure graph?